packages/api/src/controllers/access-control.ts

```diff
@@ -132,6 +138,20 @@ async function fireGateWebhook(
   return statusCode;
 }
 
+function checkRespBody(resp: string): number | null {
+  if (!resp || resp == "") {
+    return;
+  }
+  try {
+    const body = JSON.parse(resp);
+    return body.ret;
```

Comment:
I'm assuming Typescript is robust to this, but does it handle the auth server returning a blob of JSON without this field?

Reply:
Yep, tested that, it returns undefined or null and the `if(respCode)` covers it